public/assets/images/_README.md

```diff
@@ -0,0 +1,22 @@
+# How to optimise images
+
+WebP is the best format to use for most images, as it provides a good compression over PNG and JPEG. It is supported by all modern browsers (https://caniuse.com/webp) and is the recommended format for images on the web.
+
+## Using imagemagick (Recommended)
+
+This is the recommended method to convert images to WebP as it is faster and can be done in bulk.
+
+Install imagemagick from: https://imagemagick.org/script/download.php
+
+In the folder you would like to convert images, use the appropriate command:
+
+```bash
+mogrify -format webp *.png
+mogrify -format webp *.jpg
+```
+
+## Using cloudconvert
+
+If you don't want to install imagemagick, you can use cloudconvert to convert images to WebP.
+
+Go to https://cloudconvert.com/webp-converter and upload your images. You can convert multiple images at once.
```

Comment:
Cloudconvert limits the number of conversions you can do a day to 10 conversions unless you pay them. This limit applies to *all* of their tools simultaneously (so if you used 5 conversions in one tool, you now only have 5 left for every other tool)

As much as I love cloudconvert (I use them all the time) idk if we wanna shill for them like that?

Reply:
It's for this reason I listed imagemagick as most online sites have some sort of restrictions

Open to alternatives for a web converter

Reply:
https://towebp.io/ could be an option, doesn't appear to have limits

Reply:
I was in favor of just not having a cloud based option listed at all tbh, letting the user find that themselves if ImageMagick isn't an option for them (side note, iirc popular Linux distros like Ubuntu already ship with it? I never installed it but I still use it daily)

Though if you really wanted a web-based converter towebp looks fine to me. I double checked and couple find any mention of limits and they do the conversion locally which is nice

Reply:
I agree CLI is preferable, I just wanted an easy way for people to convert the images without having to install anything 

I'll switch to towebp 👍